### PR TITLE
Add kickstart plugin to extra plugins

### PR DESCRIPTION
**Pull request by Conan-Kudo**
Per discussion with @jsilhan, I am completing the migration of kickstart plugin to extras. This PR adds kickstart plugin to extras.

Reference PR: https://github.com/rpm-software-management/dnf-plugins-core/pull/124

### CODE REVIEW

File: dnf-plugins-extras.spec [hunk 1]
```diff
@@ -156,6 +164,37 @@ Obsoletes:	dnf-plugins-extras-migrate <= 0.0.4-2
...
+Provides:	dnf-command(kickstart)
+Provides:	dnf-plugins-extras-kickstart = %{version}-%{release}
+%endif
+
+%description -n python-dnf-plugins-extras-kickstart
+Kickstart Plugin for DNF, Python 2 version. Install packages listed in a
+Kickstart file.
+
+%if 0%{?fedora} >= 23
+%package -n python3-dnf-plugins-extras-kickstart
```
Comment (jsilhan) [inline dnf-plugins-extras.spec]: have you tested python3 version? IIRC the kickstart plugin was broken under python3 so we've removed it form python3 version of plugins core.

**Comment by jsilhan** [comment]
@ignatenkobrain , have this plugin in extras, please.

File: dnf-plugins-extras.spec (hunk 1 above)
Comment (Conan-Kudo) [inline dnf-plugins-extras.spec]: I forgot to push the commit that disables the plugin for python3 by default. It should be in now.

**Comment by ignatenkobrain** [comment]
Merged, thanks!

**Comment by ignatenkobrain** [comment]
@Conan-Kudo please fix tests:
```python
======================================================================
ERROR: Test whether it fails.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 79, in setUp
    base = support.BaseCliStub(self.AVAILABLE_PACKAGES, self.AVAILABLE_GROUPS)
AttributeError: 'module' object has no attribute 'BaseCliStub'

======================================================================
ERROR: Test whether it does not fail.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 79, in setUp
    base = support.BaseCliStub(self.AVAILABLE_PACKAGES, self.AVAILABLE_GROUPS)
AttributeError: 'module' object has no attribute 'BaseCliStub'

======================================================================
ERROR: Test whether it fails.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 166, in setUp
    super(KickstartCommandNotAvailableTest, self).setUp()
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 79, in setUp
    base = support.BaseCliStub(self.AVAILABLE_PACKAGES, self.AVAILABLE_GROUPS)
AttributeError: 'module' object has no attribute 'BaseCliStub'

======================================================================
ERROR: Test whether it fails if multiple arguments are given.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 92, in setUp
    super(KickstartCommandTest, self).setUp()
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 79, in setUp
    base = support.BaseCliStub(self.AVAILABLE_PACKAGES, self.AVAILABLE_GROUPS)
AttributeError: 'module' object has no attribute 'BaseCliStub'

======================================================================
ERROR: Test whether the group is installed.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 92, in setUp
    super(KickstartCommandTest, self).setUp()
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 79, in setUp
    base = support.BaseCliStub(self.AVAILABLE_PACKAGES, self.AVAILABLE_GROUPS)
AttributeError: 'module' object has no attribute 'BaseCliStub'

======================================================================
ERROR: Test whether it fails if multiple paths are given.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 92, in setUp
    super(KickstartCommandTest, self).setUp()
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 79, in setUp
    base = support.BaseCliStub(self.AVAILABLE_PACKAGES, self.AVAILABLE_GROUPS)
AttributeError: 'module' object has no attribute 'BaseCliStub'

======================================================================
ERROR: Test whether it fails if the path does not exist.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 92, in setUp
    super(KickstartCommandTest, self).setUp()
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 79, in setUp
    base = support.BaseCliStub(self.AVAILABLE_PACKAGES, self.AVAILABLE_GROUPS)
AttributeError: 'module' object has no attribute 'BaseCliStub'

======================================================================
ERROR: Test whether the package is installed.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 92, in setUp
    super(KickstartCommandTest, self).setUp()
  File "/tmp/tito/rpmbuild-dnf-plugins-extrasfxh1p3qc/BUILD/dnf-plugins-extras-0.0.10/tests/test_kickstart.py", line 79, in setUp
    base = support.BaseCliStub(self.AVAILABLE_PACKAGES, self.AVAILABLE_GROUPS)
AttributeError: 'module' object has no attribute 'BaseCliStub'
```

**Comment by Conan-Kudo** [comment]
This is fixed with #44 (which is merged in).

**Pull request closed without merging**